src/views/jobs/moderator/moderator.jsx

```diff
@@ -59,8 +59,8 @@ const Moderator = () => (
                     Interest in visual arts, programming, or teaching
                 </li>
                 <li>
-                    Not required, but would be cool: Ability to speak another
-                    language
+                    Not required, but would be cool: Ability to read and
+                    write another language.
```

Comment:
All the other bullet points do not terminate with a "."

```suggestion
                    write another language
```

Reply:
Sure! I removed the period. Good catch! @joker314